core/commands/publish.go

```diff
@@ -49,7 +50,8 @@ Publish an <ipfs-path> with another name, added by an 'ipfs key' command:
   > ipfs name publish --key=mykey /ipfs/QmatmE9msSfkKxoffpHwNLNKgwZG8eT9Bud6YoPab52vpy
   Published to QmbCMUZw6JFeZ7Wp9jkzbye3Fzp2GGcPgC3nmeUjfVF87n: /ipfs/QmatmE9msSfkKxoffpHwNLNKgwZG8eT9Bud6YoPab52vpy
 
-Alternatively, publish an <ipfs-path> using a valid PeerID(as listed by 'ipfs key list -l'):
+Alternatively, publish an <ipfs-path> using a valid PeerID(as listed by 'ipfs
+key list -l'):
 
```

Comment:
Change this two:
```
Alternatively, publish an <ipfs-path> using a valid PeerID (as listed by
'ipfs key list -l'):
```
To avoid splitting the command string (and also add a space after "PeerID")